modules/mlops/kubeflow-users/profiles/profile.template

```diff
@@ -0,0 +1,9 @@
+apiVersion: kubeflow.org/v1
+kind: Profile
+metadata:
+  name: user
+  namespace: user
+spec:
+  owner:
+    kind: User
+    name: user@amazon.com
```

Comment:
In the modules/mlops/kubeflow-users/kustomize/config.template we use  `user@example.com`. Should this be consistent?

Reply:
This gets overwritten if there are any users to be deployed as it is a template.